File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        main = {}
        for char in sentence:
            main[char] = True
        
        
        for char in "abcdefghijklmnopqrstuvwxyz":
            if char not in main:
                return False
        
        return True
                
                
            
        

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector <bool> mark(26, false);
        int index;
        
        for (int i = 0; i < sentence.size(); i++){
            if ('A' <= sentence[i] && 'Z' >= sentence[i]){
                index = sentence[i] - 'A';
            }
            else if ('a' <= sentence[i] && 'z' >= sentence[i]){
                index = sentence[i] - 'a';
            }
            else
                continue;
            
            mark[index] = true ;
            
        }
        for (int i= 0; i < 26; i++){
            if (mark[i] == false){
                return false;
            }
        
        }
        return true ;
        
    }
};